src/pages/index.js

```diff
@@ -103,101 +113,90 @@ const renCard = (data) => {
         );
         deleteCardPopup.open();
       },
+      handleLike: (card, isLiked) => {
+        cardElement.toggleLikeElement();
+        if (isLiked === true) {
+          api
+            .dislikeCard(card)
+            .then(() => {})
+            .catch((err) => {
+              console.error(err);
+            });
+        } else {
+          api
+            .likeCard(card)
+            .then(() => {})
+            .catch((err) => {
+              console.error(err);
+            });
+        }
+      },
     },
     cardSelector
   );
 
   cardSection.addItem(cardElement.getView());
 };
-// render card function
-
-const renderCard = (data) => {
-  const cardElement = new Card(
-    {
-      data,
-      handleImagePreview: (imageData) => {
-        previewImage.open(imageData);
-      },
-      handleDeleteIcon: (cardElement) => {
-        const deleteCardPopup = new PopupDelete(
-          deletePopupSelector,
-          deleteButtonSelector,
 
-          () => {
-            // console.log(cardElement.id);
+// popup with form - edit avatar
 
-            api
-              .deleteCard(cardElement)
-              .then(() => {
-                if (cardElement) {
-                  cardElement.remove();
-                }
-              })
-              .catch((err) => {
-                console.error(err);
-              });
-          }
-        );
-        deleteCardPopup.open();
-      },
-    },
-    cardSelector
-  );
-  cardSection.addItem(cardElement.getView());
-};
+const editAvatarPopUp = new PopupWithForm(formAvatarSelector, (info) => {
+  editAvatarPopUp.uploadingInfo(true);
+  api
+    .editUserAvatar(info)
+    .then((info) => {
+      setAvatar(info);
+    })
+    .catch((err) => {
+      console.error(err);
+    })
+    .finally(() => {
+      editAvatarPopUp.uploadingInfo(false);
+      editAvatarPopUp.close();
+    });
+});
 
 // //popup with form - add card
 
-const addCardPopup = new PopupWithForm(formAddSelector, (data) => {
-  const name = data.name;
-  const link = data.link;
-  const id = data._id;
-  const cardData = {
-    name,
-    link,
-    id,
-  };
-
-  newCard(cardData);
-});
+const addCardPopup = new PopupWithForm(formAddSelector, newCard);
 
 function newCard(cardData) {
```

Comment:
`newCard`

Function names should start with a **verb** in infinitive (there is such a point in the check list). It’s common in coding to do it so that functions could `speak` for themselves about what they do.



`addNewCard` or `handleCardFormSubmit`